kano/network.py

```diff
@@ -206,7 +213,12 @@ def getCellMode(s):
 
             # Parse raw data into a dictionary
             if rawdata:
-                self.data = parseRawData(rawdata)
+                try:
+                    self.data = parseRawData(rawdata)
+                except Exception as e:
+                    logger.error('unexpected error occurred while parsing rawdata {}'
+                                 .format(rawdata), exception=e)
```

Comment:
`self.data` is initialised to an empty dict at the top of the class. It should be fine.